CONTRIBUTING.md

```diff
@@ -137,5 +137,5 @@ After this, the project maintainers will review the changes and will merge your
 
 ## 🌐 Web3Community
 
-- 😕 Not sure where to start? Join our community on [Discord](https://discord.gg/37QFQ7J78B)\
+- 😕 Not sure where to start? Join the Web3Community [Discord](https://discord.gg/TSRwqx4K2v)
```

Comment:
Dev Protocol Community not ~~Web3Community~~

Reply:
> Dev Protocol Community not ~Web3Community~

Why does the header say Web3Community? 🤔

Reply:
yep xD